consensus/istanbul/backend/announce.go

```diff
@@ -114,9 +120,7 @@ func (sb *Backend) announceThread() {
 				// hence more likely that they will be aware that this node is
 				// within that set.
 				time.AfterFunc(1*time.Minute, func() {
```

Comment:
Suggest getting rid of this function call as it will get announced in one minute as part of the event loop anyway. This only serves to have it announce twice quickly.